instrumentation/runtime-telemetry/runtime-telemetry-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/RuntimeMetrics.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.runtimemetrics.java8;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.runtimemetrics.java8.internal.JmxRuntimeMetricsUtil;
+import java.io.Closeable;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/** The entry point class for runtime metrics support using JFR and JMX. */
+public final class RuntimeMetrics implements Closeable {
+
+  private static final Logger logger = Logger.getLogger(RuntimeMetrics.class.getName());
+
+  private final AtomicBoolean isClosed = new AtomicBoolean();
+  private final List<AutoCloseable> observables;
+
+  RuntimeMetrics(List<AutoCloseable> observables) {
+    this.observables = Collections.unmodifiableList(observables);
+  }
+
+  /**
+   * Create and start {@link RuntimeMetrics}.
+   *
+   * <p>Listens for select JFR events, extracts data, and records to various metrics. Recording will
```

Comment:
I think this javadoc should be adjusted. JFR recording and metrics are only with JDK17+. This is because the JFR streaming feature only was introduced in JDK 14.